app/src/main/java/org/greenstand/android/TreeTracker/database/AppDatabase.kt

```diff
@@ -34,14 +34,14 @@ abstract class AppDatabase : RoomDatabase() {
                                                     AppDatabase::class.java,
                                                     DB_NAME
                     )
-                        .addMigrations(MIGRATION_1_2)
+                        .addMigrations(MIGRATION_3_4)
                         .build()
                 }
             }
             return INSTANCE!!
         }
 
-        private const val DB_NAME = "treetracker.v1.db"
+        private const val DB_NAME = "treetracker.v2.db"
```

Comment:
Why was this changed?

Reply:
The older db is named `v2.db`. Since we are migrating the db, I believe it is better to keep the name the same. This would keep the variations to a minimum and keep the migration script less complicated.